pcsx2/IopBios.cpp

```diff
@@ -110,832 +114,898 @@ namespace R3000A {
+								n++;
+								break;
+
+							case 'p':
+							case 'i':
+							case 'd':
+							case 'D':
+							case 'o':
+							case 'O':
+							case 'x':
+							case 'X':
+								ptmp += sprintf(ptmp, tmp2, (u32)iopMemRead32(sp + n * 4));
+								n++;
+								break;
+
+							case 'c':
+								ptmp += sprintf(ptmp, tmp2, (u8)iopMemRead32(sp + n * 4));
+								n++;
+								break;
 
-						case 's':
+							case 's':
 							{
 								std::string s = iopMemReadString(iopMemRead32(sp + n * 4));
 								ptmp += sprintf(ptmp, tmp2, s.data());
 								n++;
 							}
 							break;
 
-						case '%':
-							*ptmp++ = fmt[i];
-							break;
+							case '%':
+								*ptmp++ = fmt[i];
+								break;
 
-						default:
-							break;
-					}
-					i++;
-					break;
+							default:
+								break;
+						}
+						i++;
+						break;
 
-				default:
-					*ptmp++ = fmt[i++];
-					break;
+					default:
+						*ptmp++ = fmt[i++];
+						break;
+				}
 			}
+			*ptmp = 0;
+			iopConLog(ShiftJIS_ConvertString(tmp, 1023));
+
+			return 1;
 		}
-		*ptmp = 0;
-		iopConLog( ShiftJIS_ConvertString(tmp, 1023) );
+	} // namespace sysmem
 
-		return 1;
-	}
-}
+	namespace loadcore
+	{
+		void RegisterLibraryEntries_DEBUG()
+		{
+			const std::string modname = iopMemReadString(a0 + 12);
+			DevCon.WriteLn(Color_Gray, "RegisterLibraryEntries: %8.8s version %x.%02x", modname.data(), (unsigned)iopMemRead8(a0 + 9), (unsigned)iopMemRead8(a0 + 8));
+		}
+	} // namespace loadcore
 
-namespace loadcore {
-	void RegisterLibraryEntries_DEBUG()
+	namespace intrman
 	{
-		const std::string modname = iopMemReadString(a0 + 12);
-		DevCon.WriteLn(Color_Gray, "RegisterLibraryEntries: %8.8s version %x.%02x", modname.data(), (unsigned)iopMemRead8(a0 + 9), (unsigned)iopMemRead8(a0 + 8));
-	}
-}
+		static const char* intrname[] = {
+			"INT_VBLANK", "INT_GM", "INT_CDROM", "INT_DMA",            //00
+			"INT_RTC0", "INT_RTC1", "INT_RTC2", "INT_SIO0",            //04
+			"INT_SIO1", "INT_SPU", "INT_PIO", "INT_EVBLANK",           //08
+			"INT_DVD", "INT_PCMCIA", "INT_RTC3", "INT_RTC4",           //0C
+			"INT_RTC5", "INT_SIO2", "INT_HTR0", "INT_HTR1",            //10
+			"INT_HTR2", "INT_HTR3", "INT_USB", "INT_EXTR",             //14
+			"INT_FWRE", "INT_FDMA", "INT_1A", "INT_1B",                //18
+			"INT_1C", "INT_1D", "INT_1E", "INT_1F",                    //1C
+			"INT_dmaMDECi", "INT_dmaMDECo", "INT_dmaGPU", "INT_dmaCD", //20
+			"INT_dmaSPU", "INT_dmaPIO", "INT_dmaOTC", "INT_dmaBERR",   //24
+			"INT_dmaSPU2", "INT_dma8", "INT_dmaSIF0", "INT_dmaSIF1",   //28
+			"INT_dmaSIO2i", "INT_dmaSIO2o", "INT_2E", "INT_2F",        //2C
+			"INT_30", "INT_31", "INT_32", "INT_33",                    //30
+			"INT_34", "INT_35", "INT_36", "INT_37",                    //34
+			"INT_38", "INT_39", "INT_3A", "INT_3B",                    //38
+			"INT_3C", "INT_3D", "INT_3E", "INT_3F",                    //3C
+			"INT_MAX"                                                  //40
```

Comment:
Could you undo the clang-format on this one and add `// clang-format off` and `// clang-format on` comments around it?

Reply:
Done!

Reply:
You forgot the "undo clang format" part